app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +23,24 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".activities.DictionaryActivity"
+            android:label="@string/title_activity_dictionary"
+            android:parentActivityName=".activities.AddNewWordActivity"
+            android:theme="@style/MaterialTheme">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value=".activities.AddNewWordActivity" />
```

Comment:
The android sample contains a bunch of things that are not necessarily useful for a given application. For example, the meta-data field is not reallly utilised in your app, so I would suggest removing them (less typing ;) )

http://developer.android.com/guide/topics/manifest/meta-data-element.html